campos_potenciais2.py

```
@@ -0,0 +1,278 @@
+    print("goal encontrado na cena; usando posição do goal:", x_goal_world)
+
+# ----  TRANSFORMAR leitura dos sensores para o frame DO ROBO ----
+def proximity_detect_in_robot_frame(sensorHandle):
+    """
+    Lê proximity sensor; se detectou, transforma o ponto detectado -> frame do robô.
+    Retorna (detected, obs_in_robot (x,y,z), dist) ou (False, None, None).
+    """
+    try:
+        # readProximitySensor costuma retornar (state, detectedPoint)
+        state, detectedPoint = sim.readProximitySensor(sensorHandle)
+    except Exception:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)